### ISSUE TITLE
swift support?

### ISSUE THREAD

**Issue by connectdotz** [travisjeffery/ClangFormat-Xcode#55]
Is there any plan to support swift source code formating?

**Comment by haaakon** [comment]
By the way, i asked on the Clang forum in june about this, but got no response (70 views on the question) http://clang-developers.42468.n3.nabble.com/Will-clang-format-support-the-format-of-Swift-td4039910.html

**Comment by cyril94440** [comment]
+1

**Comment by cybertk** [comment]
+1

**Comment by travisjeffery** [comment]
yeah no amount of +1s here is gonna change anything, send your +1s to the clang mailing list if anything. locking this thread
